server/pbenchinacan/deploy

```diff
@@ -60,20 +60,31 @@ else
 fi
 
 # Update the Nginx favicon and provide some static fallback pages
+#
+# NOTE:  this is the first invocation of the Podman run command in this script,
+#        so we apply the pull policy here; after this invocation (if it doesn't
+#        fail...), the container image will be present in the local repository,
+#        and so we use the default policy ("missing") for the other invocations.
+#        (Otherwise, we risk pulling a remote image when we shouldn't.)
```

Comment:
It's a little odd to see this extended comment about `podman run` before the `podman unshare` command ...

Reply:
Yeah...that's a lesser of weevils problem...but I'll try to address it after #3270 lands, at which point it might be moot.

Reply:
OK, I've re-implemented this change, now that there are more and different Podman `run` invocations...so, I believe this comment is in fact now moot.